ros_cross_compile/pipeline_stages.py

```diff
@@ -28,3 +32,16 @@
                                          ('custom_script', Optional[Path]),
                                          ('custom_data_dir', Optional[Path]),
                                          ('custom_setup_script', Optional[Path])])
+
+
+class PipelineStage(metaclass=ABCMeta):
```

Comment:
nitpick: This might be cleaner-looking as `PipelineStage(ABC)` - https://docs.python.org/3/library/abc.html#abc.ABC